antidox/perspective_test.py

```diff
@@ -185,7 +184,50 @@ def test_contains_toxicity_false(self):
+    However, ''The Neighborhoods of Brooklyn'' shows the boundary at Park Place, putting the square (just) in Bed-Stuy;
+    Then again, the ''AIA Guide to New York City'' uses the Atlantic Avenue border;
+    The NYCLPC's Crown Heights North Historic District has a northern boundary which is midblock south of Atlantic Avenue, meaning it, too, recognizes those blocks as being part of Crown Heights. (Grant Square itself is not part of the district, but the building at the southeast corner of Dean and Rogers, confusingly called the "Fort Greene Grant Square Center", even though it's nowhere near Fort Greene, is in the district);
+    The website of the Washington Temple Church of God in Christ says it is "Located in the Crown Heights section of Brooklyn".
+    All-in-all, the usage appears to be very much on the side of Grant Square being in Crown Heights and not in Bed-Stuy - which brings up the question, where did you see the C.O.G.I.C described as being in Bed-Stuy?   
+    JimHenderson's picture of the church. I have two of my own coming. Do you know what else I noticed? Google Maps also shows the western border of Crown Heights as being along Washington Avenue, which would place ''both'' Studebaker Buildings (Bedford Avenue and Sterling Place, as well as Dean Street) as being in Crown Heights! -   
+     Jim often takes his photos while on group bicycle rides, maybe he didn't realize he had crossed over a neighborhood boundary.The NYCLPC clearly describes the Studebaker Building on Bedford as in Crown Heights.  The other one isn't a landmark, but a lot of sites describe it as being in Crown Heights as well   and by the definitions of pretty most all of the sources above too.  It's really too far south to be in Bed-Stuy, so I wonder why that designation got started?   
+    I believe it was the former Franklin Shuttle Station next door. Well, since both are officially in Crown Heights, we really need better names for both of them now.    
+     I shan’t claim to know; sometimes I’m not even sure whether I’ve crossed into Queens.   
+     Don't you see the "You are now leaving Brooklyn - Fugeddabouddit" signs?   
+     In his defense, those signs can't be at every street that crosses the Brooklyn-Queens line. As far as the Studebaker Buildings, since both are officially considered to be in Crown Heights, how about renaming the gallery for the landmark "Studebaker Building (Bedford Avenue, Brooklyn)," and I'll rename the other one "Studebaker Building (Dean Street, Brooklyn)?" I'll even correct Dean Street (BMT Franklin Avenue Line).-   
+
+    Buzz Aldrin in project scope?
+    Why would Buzz Aldrin be in  scope?    
+    No idea. I've removed it.   
+     WP 1.0 Bot Beta 
+    Error: <HttpError 400 when requesting https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=AIzaSyA5kva-B5eLUBMD-VB426Pan3AlPXzWOXc&alt=json returned "Attribute THREAT does not support request languages: hu">
```

Comment:
I don't think this Perspective string (or maybe the comments above about Buzz Aldrin) was in the Wikipedia response and may have gotten pasted here by accident, you can remove these